forward43/api_ulule.py

```diff
@@ -8,7 +8,7 @@
 # %%   Import packages
 import json
 import math
-import pandas as pd
+# import pandas as pd
```

Comment:
perfect opportunity to try out a `fixup!` commit :D

Reply:
Merged this commit with next one